test/linters/linters.test.js

```diff
@@ -69,7 +71,7 @@ afterAll(async () => {
 // Test all linters
 describe.each(linterParams)(
 	"%s",
-	(projectName, linter, commandPrefix, extensions, getLintParams, getFixParams) => {
+	(projectName, linter, commandPrefix, extensions, args, getLintParams, getFixParams) => {
```

Comment:
Not fond of how adding `args` affected all other tests.
Needed to add this as clippy by default will avoid fixing if the repo has local changes, and the other tests happen to leave the repo with local changes.